src/python/platforms/android/symbols_downloader.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Used to download Android symbols."""
+from __future__ import print_function
+
+import os
+
+from base import utils
+from google_cloud_utils import storage
+from metrics import logs
+from platforms.android import fetch_artifact
+from platforms.android import settings
+from system import archive
+from system import environment
+from system import shell
+
+
+def get_kernel_info():
+  """Get the build_id, target, and symbols_archive_filename."""
+  build_id = settings.get_kernel_build_id()
+  target = settings.get_kernel_name()
+  if not build_id or not target:
+    return None, None, None
+  symbols_archive_filename = '%s-%s-repo.prop' % (target, build_id)
```

Comment:
symbol archive filename should be a different function, that just take target, build_id.
and then get_kernel_info should in settings.py as well